src/OSPSuite.R/Services/PopulationTask.cs

```diff
@@ -1,20 +1,30 @@
 ﻿using System.Collections.Generic;
 using System.Data;
+using System.IO;
 using System.Linq;
 using OSPSuite.Core.Domain;
 using OSPSuite.Core.Domain.Populations;
 using OSPSuite.Core.Domain.Services;
 using OSPSuite.Core.Extensions;
 using OSPSuite.Infrastructure.Import.Services;
 using OSPSuite.R.Extensions;
+using OSPSuite.Utility;
 using OSPSuite.Utility.Extensions;
 
 namespace OSPSuite.R.Services
 {
    public interface IPopulationTask
    {
       IndividualValuesCache ImportPopulation(string fileFullPath);
-      DataTable PopulationTableFrom(IndividualValuesCache population, IModelCoreSimulation simulation);
+
+      DataTable PopulationTableFrom(IndividualValuesCache population, IModelCoreSimulation simulation = null);
+
+      /// <summary>
+      /// Loads the population from the <paramref name="populationFile"/> and split the loaded population according to the <paramref name="numberOfCores"/>.
+      /// Resulting files will be exported in the <paramref name="outputFolder"/>. File names will be constructed using the <paramref name="outputFileName"/> concatenated with the node index.
+      /// Returns an array of string containing the full path of the population files created
+      /// </summary>
+      IReadOnlyList<string> SplitPopulation(string populationFile, int numberOfCores, string outputFolder, string outputFileName);
```

Comment:
This method will split a population from a file in multiple files. Exactly what is needed for // in R. No need to implement it twice